### DISCUSSION TITLE
What is the relationship between GitHub Advisory Database and Trivy Vulnerability Data Source?

### DISCUSSION THREAD

**Discussion by fighterhit** [aquasecurity/trivy#1910] [Q&A] [answered]
What is the relationship between [GitHub Advisory Database](https://github.com/advisories) and [Trivy Vulnerability Data Source](https://aquasecurity.github.io/trivy/v0.25.0/vulnerability/detection/data-source/)? For example, I have a springboot program that contains a vulnerability. Since it is a Java program, when trivy scans the program's docker image for vulnerabilities, will it **only** use the [GitLab Advisories Community](https://gitlab.com/gitlab-org/advisories-community) and [GitHub Advisory Database (Maven)](https://github.com/advisories?query=ecosystem%3Amaven) listed in the [Trivy Vulnerability Data Source](https://aquasecurity.github.io/trivy/v0.25.0/vulnerability/detection/data-source/)? But I see there is also a [NVD](https://nvd.nist.gov/) in the data source, will this be used when scanning my springboot program? What does it do? Thanks!

![image](https://user-images.githubusercontent.com/23624910/161191548-080c1612-c6c6-4240-a9a7-7a651b9a1e83.png)

**Comment by DmitriyLewen** [comment, marked as answer]
Hello @fighterhit 
Thanks for your interest to Trivy.
Trivy use only [GitLab Advisories Community](https://gitlab.com/gitlab-org/advisories-community) and [GitHub Advisory Database (Maven)](https://github.com/advisories?query=ecosystem%3Amaven) for scan Java files.

Regards, Dmitriy

**Reply by fighterhit** [reply]
Thank you @DmitriyLewen , when will NVD be used?

**Reply by DmitriyLewen** [reply]
we are not working on it at the moment

**Reply by fighterhit** [reply]
ok, thank you!
